camel/bots/discord/discord_app.py

```diff
@@ -53,6 +62,9 @@ def __init__(
             token (Optional[str]): The Discord bot token for authentication.
                 If not provided, the token will be retrieved from the
                 environment variable `DISCORD_TOKEN`.
+            client_id (Optional[str]): The client ID for Discord OAuth.
```

Comment:
same as other variables

```suggestion
            client_id (Optional[str]): The client ID for Discord OAuth. (default: :obj:`None`)
```